### ISSUE TITLE
Farbkontrast bei Buttons nicht ausreichend

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/rrze-elements#313]
Der Farbkontrast bei einer gegebenen Buttonfarbe zur Schriftfarbe ist nicht ausreichend. Es muss für die höheren Anfordeurngen auf 7:1 gestellt sein.

Vgl: 
<img width="2110" alt="grafik" src="https://github.com/RRZE-Webteam/rrze-elements/assets/1070669/4a6e3c6a-d1f6-476c-86ad-4f02dca2b6b1">

https://my2.siteimprove.com/Inspector/764733/A11Y/Page?pageId=64788178017&impmd=1A3574CC7C3B5ABC4A1CA70A1865D7A2&conf=a+aa+aaa+aria#/sia-r66/failed/Qzd2VENTbm0=;YzgyODRjMzQ=

**Comment by cassandre** [comment]
#AB05F2 / #FFFFFF => 5,2
#AB05F2 / #000000 => 4,03
Sollen dann Farben, die keine ausreichenden Kontrast ermöglichen, deaktiviert werden? 
Oder dunkler/heller gerechnet?
Oder ein Warnung ausgegeben, die nur angemeldet sichtbar ist? ...
